### DISCUSSION TITLE
Teleport the person into the car I'm driving

### DISCUSSION THREAD

**Discussion by Marlodor96** [YimMenu/YimMenu#1796] [Q&A]
it's possible to add "bring person into the car I'm driving" mod?
Thanks

**Comment by Marlodor96** [comment]
maybe it can be like this ?

#include "backend/player_command.hpp"
#include "backend/command.hpp"
#include "natives.hpp"
#include "util/teleport.hpp"

namespace big
{
	class teleport_player_in_your_last_vehicle : player_command;          rem OR ": command;"
	{
		using player_command::player_command;                               rem OR "using command::command;"

		virtual void execute(const std::vector<uint64_t>&, const std::shared_ptr<command_context> ctx)
		{
			if (g_local_player && g_local_player->m_vehicle)
			{
				const Vehicle veh = g_pointers->m_gta.m_ptr_to_handle(player->m_vehicle);
				teleport::bring_player(veh);
			}
		}
	};

	teleport_player_in_your_last_vehicle g_teleport_player_in_your_last_vehicle("lastvehtp", "Teleport Player inside your Last Vehicle", "Teleports player into your last driven vehicle", 0);
}

**Reply by Yimura** [reply]
That would put a vehicle as a passenger of another vehicle, this will not work lol.

**Reply by Marlodor96** [reply]
yes but maybe u can gift the car with this method.
like the replace car in garage glitch

**Reply by Yimura** [reply]
At the best this will do nothing, at the worst your game will crash.

**Reply by Marlodor96** [reply]
it is only an example, im not sure if it work or not, but probably not lol
